src/js/media-fullscreen-button.js

```diff
@@ -20,30 +20,69 @@ class MediaFullscreenButton extends MediaChromeButton {
     super();
 
     this.icon = enterFullscreenIcon;
+  }
 
-    this.addEventListener("click", e => {
-      const media = this.media;
-
-      if (this.mediaChrome == document.fullscreenElement) {
-        document.exitFullscreen();
-      } else {
-        if (document.pictureInPictureElement) {
-          // Should be async
-          document.exitPictureInPicture();
-        }
+  onClick() {
```

Comment:
Moved the button click functionality to the `onClick` method provided by the base class.